apteryx.c

```diff
@@ -664,7 +666,17 @@ apteryx_set_full (const char *path, const char *value, uint64_t ts, bool ack)
 }
 
 bool
-apteryx_cas_string (const char *path, const char *key, const char *value, uint64_t ts)
+apteryx_set_full (const char *path, const char *value, uint64_t ts, bool ack)
+{
+    set_flags flags = FLAG_SET_WATCH;
```

Comment:
int flags - I'm not comfortable with not-enumerated values showing as a "set_flags"